bbn-1/pops/README.md

```diff
@@ -0,0 +1,422 @@
+    "btcSignBaby": "AkcwRAIgcrI2IdD2JSFVIeQmtRA3wFjjiy+qEvqbX57rn6xvWWECIDis7vHSJeR8X91uMQReG0pPQFFLpeM0ga4BW+Tt2V54ASEDefcQA1iRWLJXk0VUCwi7x0l0xJ3V4HguMdDeZ0VA1RM=",
+    "babySignBtc": "FnYTm9ZbhJZY202R9YBkjGEJqeJ/n5McZBpGH38P2pt0YRcjwOh8XgoeVQTU9So7/RHVHHdKNB09DVmtQJ7xtw==",
+    "babyPublicKey": "Asezdqkvh+kLbuD75DirSwi/QFbJjFe2SquiivMaPS65"
+}
+```
+
+This command will create a JSON file that contains the pop result to the
+specified output path. It has several flag options:
+
+- `--btc-address` is the address of the BTC key used when staking in
+  phase-1. Bech-32 encoded.
+- `--baby-address` is the address of the BABY key created
+  in [step-1](#1-create-babylon-keys-and-addresses). Bech-32 encoded.
+- `--keyring-dir` is the directory of the keyring created
+  in [step-1](#1-create-babylon-keys-and-addresses).
+- `--keyring-backend` is the backend of the keyring created
+  in [step-1](#1-create-babylon-keys-and-addresses).
+- `--btc-wallet-host` is the host of the BTC wallet, default for BTC
+  mainnet is `localhost:8332`.
+- `--output-file` specifies the file path of the raw JSON result. Note
+  that if the flag is not specified, the JSON file won't be generated.
+
+This command will produce json output:
+
+```bash
+{
+    "babyAddress": "bbn1xjz8fs9vkmefdqaxan5kv2d09vmwzru7jhy424",
+    "btcAddress": "bc1qcpty6lpueassw9rhfrvkq6h0ufnhmc2nhgvpcr",
+    "btcPublicKey": "79f71003589158b2579345540b08bbc74974c49dd5e0782e31d0de674540d513",
+    "btcSignBaby": "AkcwRAIgcrI2IdD2JSFVIeQmtRA3wFjjiy+qEvqbX57rn6xvWWECIDis7vHSJeR8X91uMQReG0pPQFFLpeM0ga4BW+Tt2V54ASEDefcQA1iRWLJXk0VUCwi7x0l0xJ3V4HguMdDeZ0VA1RM=",
+    "babySignBtc": "FnYTm9ZbhJZY202R9YBkjGEJqeJ/n5McZBpGH38P2pt0YRcjwOh8XgoeVQTU9So7/RHVHHdKNB09DVmtQJ7xtw==",
+    "babyPublicKey": "Asezdqkvh+kLbuD75DirSwi/QFbJjFe2SquiivMaPS65"
+}
+```
+
+where:
+
+- `babyAddress` is the address of the BABY key created in step 1.
+  Bech-32 encoded.
+- `btcAddress` is the address of corresponding to the BTC key used
+  when staking in phase-1. Bech-32 encoded.
+- `btcPublicKey` is the staker public key used. Hex encoded.
+- `btcSignBaby` is the BI322 signature made by `btcAddress`. Base64 encoded.
+- `babySignBtc` is the ADR36 signature made by `babyPublicKey`. Base64 encoded.
+- `babyPublicKey` is the Babylon public key, corresponding to
+  the `babyAddress`. Base64 encoded.
+
+### 3.4. Validate the Proof of Possession (PoP)
+
+To validate the JSON file that contains the PoP,
+run `stakercli pop validate` command:
+
+```bash
+ $ stakercli pop validate /path/to/pop.json
+ Proof of Possession is valid!
+```
+
+If `Proof of Possession is valid!` is shown, the pop is successfully validated.
+
+## 4. Create Pull Request
+
+The PoP JSON files for finality providers and stakers are stored under the
+`fps` and `stakers` directories respectively. Each file should be identified
+by a unique name, e.g., `${btc_pk_pop}.json`, with `.json` extension.
```

Comment:
are we going to accept PoPs as input for PR in this repository?

Reply:
Yes